website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx

```diff
@@ -34,7 +34,12 @@ To join an organization, you must be invited by one of its [owners][] or a user
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-You can leave an organization from your user account settings. Refer to [Account Settings: Organizations](/terraform/cloud-docs/users-teams-organizations/users#organizations) for details.
+To leave an organization:
+
+1. Click the Terraform logo in the upper left corner to navigate to the **Organizations** page.
+1. Click the **ellipses (...)** next to the organization and select **Leave organization**. 
```

Comment:
I don't think we have a standard way to do this, but I think `Click the ellipses (**...**)` is a little clearer because the bolded thing is exactly the thing you look for on the page.